modules/KIWILinuxRC.sh

```diff
@@ -9219,19 +9236,24 @@ function setupHybridCowDevice {
         Echo "Failed to mount hybrid persistent filesystem !"
         return 1
     fi
-    if [ ! -e "/cow/cowfile" ];then
-        local cowsize=$(($(blockdev --getsize64 $hybrid_device) / 2))
-        local exception_handling="false"
-        qemu-img create /cow/cowfile $cowsize
-        if ! createFilesystem \
-            "/cow/cowfile" "" "" "" $exception_handling "btrfs"
-        then
-            Echo "Failed to create hybrid persistent cow filesystem"
-            return 1
-        fi
-    fi
-    export HYBRID_RW=$(loop_setup /cow/cowfile)
-    if [ ! -e "$cowdevice" ];then
+    if [ ! -e "$hybrid_filename" ];then
+	# default filesize is half of partition's capacity
+        local cowsize="$(($(blockdev --getsize64 $hybrid_device) / 2))"
+        # but not for FAT
+        if [ $hybrid_fs = "fat" ];then
+	        if [ $cowsize -gt "4294967295" ];then
+		        # filesize is reduced to its limitation
+		        cowsize=4294967295
+	        fi
+        fi
+        # user can override it on boot
+	if [ ! -z "$kiwi_hybridpersistent_filesize" ];then
```

Comment:
maybe I overlooked it, but I see you have changed the schema to provide an attribute for this but I don't see the code evaluating the XML and adding kiwi_hybridpersistent_filesize as part of the profile environment so that it can be actually read